package.json

```diff
@@ -50,6 +50,7 @@
     "tailwindcss": "yarn:@tailwindcss/postcss7-compat@^2.1.2"
   },
   "dependencies": {
+    "@streamlinehq/streamlinehq": "^3.0.3",
```

Comment:
their icons are great, but we need to mind the [license terms](https://www.notion.so/Streamline-Free-License-5b7339ddeb194dc8bb1b0e97aadc011f)

if decided, we can acquire the [premium license](https://www.notion.so/Streamline-Premium-License-af70bce755d14e06b78010249468530e)

Reply:
but i guess the other question is, where is this being used?

Reply:
oh i was using that NPM package to download the assets locally, then ended up not using it cuz it had troubles with applying styles

in terms of license @sallkall @YizhiCatherineZhang are we premium subscribers already?

Reply:
We are on a premium subscription (icons only no illustrations). Ex: the npm token I gave @hyx131 is a premium-only feature.